community/cypher/compatibility-suite/src/test/java/cypher/FeatureSuiteTest.java

```diff
@@ -49,14 +49,15 @@
     {
     }
 
-    @RunWith( Cucumber.class )
-    @CucumberOptions( plugin = {
-            "pretty", "html:target/cost-compiled",
-            "cypher.cucumber.reporter.CypherResultReporter:target/cost-compiled",
-            "cypher.cucumber.db.DatabaseProvider:target/dbs",
-            "cypher.cucumber.db.DatabaseConfigProvider:/cypher/db/config/cost-compiled.json",
-    } )
-    public static class CostCompiled
-    {
-    }
+    // The compiled runtime is not officially supported, and it fails on running these tests
+//    @RunWith( Cucumber.class )
+//    @CucumberOptions( plugin = {
+//            "pretty", "html:target/cost-compiled",
+//            "cypher.cucumber.reporter.CypherResultReporter:target/cost-compiled",
+//            "cypher.cucumber.db.DatabaseProvider:target/dbs",
+//            "cypher.cucumber.db.DatabaseConfigProvider:/cypher/db/config/cost-compiled.json",
+//    } )
+//    public static class CostCompiled
+//    {
+//    }
```

Comment:
We should probably just remove this code. It can be retrieved from the history whenever we start working on the runtime again.